packages/share_plus/share_plus/android/src/main/kotlin/dev/fluttercommunity/plus/share/Share.kt

```diff
@@ -71,9 +71,9 @@ internal class Share(
                 PendingIntent.getBroadcast(
                     context,
                     0,
-                    Intent(ShareSuccessManager.BROADCAST_CHANNEL),
+                    Intent(context, SharePlusPendingIntent::class.java),
                     PendingIntent.FLAG_UPDATE_CURRENT or immutabilityIntentFlags
-                ).intentSender
+                ).getIntentSender()
```

Comment:
Let's stick with Kotlin getter access syntax as it was before.

Reply:
Actually didn't know that this was a feature of Kotlin - thank you :)